frontend/src/components/tables/Columns/forms/FormResponseColumns.tsx

```
@@ -47,17 +69,98 @@ export const useFormResponseColumns = () =>
-        cell: ({ row }) => formatSubmittedAt(row.original.submittedAt),
+        id: "actions",
+        cell: ({ table, row }) => {
+          const response = row.original;
+          const handleDeleteResponse = table.options.meta?.handleDeleteData;
+          const handleViewResponse = table.options.meta?.handleViewData;
+          const [isDeleteModalOpen, setIsDeleteModalOpen] = useState(false);
+
+          return (
+            <>
+              <DropdownMenu dir="rtl">
+                <DropdownMenuTrigger asChild>
```

Biome (2.3.13)

[error] This hook is being called from a nested function, but all hooks must be called unconditionally from the top-level component.

For React to preserve state between calls, hooks needs to be called unconditionally and always in the same order.
See https://reactjs.org/docs/hooks-rules.html#only-call-hooks-at-the-top-level

(lint/correctness/useHookAtTopLevel)